var/spack/repos/builtin/packages/qt/package.py

```diff
@@ -250,6 +250,8 @@ class Qt(Package):
     else:
         conflicts('platform=darwin', when='@:4.8.6',
                   msg="QT 4 for macOS is only patched for 4.8.7")
+        conflicts('target=aarch64', when='@:5.15.3',
```

Comment:
Does this apply to all aarch64 targets or `aarch64` itself? I think it should be `aarch64:`.

Also should this be gated by `darwin` also, or does it also apply on linux?

Reply:
Ahh, the colon is what I'm missing. Thanks. The `else` here means it applies only to darwin.